search/geocoder.cpp

```diff
@@ -524,11 +556,18 @@ void Geocoder::GoImpl(vector<shared_ptr<MwmInfo>> & infos, bool inViewport)
                        m_cities = citiesFromWorld;
                      });
 
+      if (m_params.IsCategorialRequest())
+      {
+        bool const aroundPivot = index < numIntersectingMaps;
```

Comment:
Please, move this out of the if and reuse in both branches.

Reply:
done